zml/pjrtx.zig

```diff
@@ -145,7 +196,8 @@ pub const Event = opaque {
         return self.inner().getEventError(api);
     }
 
-    pub fn await_(self: *Event, api: *const Api) !void {
+    pub const await_ = awaitt;
```

Comment:
can we chose a syntax and stick to it, I feel we flip flop every two month on that